backend/globaleaks/security.py

```diff
@@ -430,6 +430,22 @@ def destroy_environment(self):
             log.err("Unable to clean temporary PGP environment: %s: %s" % (self.gnupg.gnupghome, excep))
 
 
+def encrypt_pgp_message(pgp_key_public, pgp_key_fingerprint, msg):
+    gpob = GLBPGP()
+
+    try:
+        gpob.load_key(pgp_key_public)
+        body = gpob.encrypt_message(pgp_key_fingerprint, msg)
+    except Exception as excep:
+        log.err("Error in PGP interface object: %s" % excep)
```

Comment:
i suppose this code is just a copy of other code.

i identify here a unneded complexity as the exception is printed and raised (that would anyhow print it as unhandled exceptio. i would so avoid to print it here.

the code could be rewritten simply ad:
```
except:
    raise
```

in addition the log.err as discussed should try to be written with log.err(string, arg1, arg2, arg3) in order to avoid creating the string at runtime if the the debug is disabled.